### PR TITLE
Add underline bug fix

### PR DESCRIPTION
**Pull request by joemoongit**
Please let me know if you have any suggestions.

### CODE REVIEW

**Comment by julianhille** [comment]
Just some notes before i get to test it:
- the lineWidth is missing in the d.ts file as optional member of the WriteTextOptions interface
- also a changelog entry is missing.

**Comment by joemoongit** [comment]
I've made the changes

**Comment by julianhille** [comment]
Thank you for your contribution and effort. Will have a look (hopefully) this evening, as im really busy at work and in private actually but anyway will take some time for it as far as there is free time.

**Comment by julianhille** [comment]
are you sure that this is working?
i'm locally unable to see a difference or find the way how to use that. could you add a sample here for me so that i can see a difference between a lineWidth of `2` and `10000` or so.

**Comment by joemoongit** [comment]
so i've passed in some text containing ```<u>``` tags into text()
here are the results when i use lineWidth's default value, when i set it to 2, and when i set it to 10000

[pdf-lineWidth-default.pdf](https://github.com/julianhille/MuhammaraJS/files/11022330/pdf-lineWidth-default.pdf)
[pdf-lineWidth-2.pdf](https://github.com/julianhille/MuhammaraJS/files/11022332/pdf-lineWidth-2.pdf)
[pdf-lineWidth-10000.pdf](https://github.com/julianhille/MuhammaraJS/files/11022335/pdf-lineWidth-10000.pdf)

it looks like lineWidth is being set in makeTextObjects(), and i am simply using that value when underlining the text in addUnderline()

**Comment by julianhille** [comment]
Will Test. I did test it totally different and apparently completely wrong.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+- Underline in text object
+
+### Fixed
+
 - Recipe type / arg documentation
```
Comment (julianhille) [inline CHANGELOG.md]:
```suggestion
- Underline in text object
- Recipe type / arg documentation
```

**Review by julianhille — APPROVED** [review]

**Pull request merged**